android/MobiledgeXSDKDemo/build.gradle

```diff
@@ -44,7 +44,7 @@ allprojects {
                 username artifactory_user
                 password artifactory_password
             }
-            url "https://artifactory.mobiledgex.net/artifactory/maven-releases/"
+            url "https://artifactory.mobiledgex.net/artifactory/maven-development/"

```

Comment:
Will change back to "releases" when SDK is published there.

Reply:
ok